### PR TITLE
13 line chart assets depreciation

### PR DESCRIPTION
**Pull request by ivangarcilazo**

### CODE REVIEW

**Comment by mohitkhatri06** [comment]
image
@ivangarcilazo I am not able to see the chart. let me know if I am missing somthing

**Comment by ivangarcilazo** [comment]
It may have something to do with the environments? It works correctly for me, please let me see if it's me... Let me know if it's working again or something.

File: src/provider/Provider.js [hunk 1]
```diff
@@ -334,6 +335,23 @@ const Provider = ({ children }) => {
                   payload: finalMonthlyExp,
                });
             });
+            
+         await axios
```
Comment (mohitkhatri06) [inline src/provider/Provider.js:339]: this is giving error as axios is not there.

Comment (ivangarcilazo) [reply src/provider/Provider.js:339]: Okay Mohit, thanks... I'll check

**Comment by mohitkhatri06** [comment]
bit spacing in between the text. And @bharathkumarchowdary we should use the number as 10,00,000 instead of 1M?
image

**Comment by bharathkumarchowdary** [comment]
> bit spacing in between the text. And @bharathkumarchowdary we should use the number as 10,00,000 instead of 1M?
> image

@mohitkhatri06 its better visually to use multiples instead of full number

File: src/components/Charts/AssetsDepreciationsChart.jsx [hunk 2]
```diff
@@ -48,10 +49,15 @@ function shortDate (date){
 
 
 
-export default function AssetsDeprecationsChart({data}){
+export default function AssetsdepreciationChart({data}){
     const brushRef = useRef(undefined)
-   
-    if(data[0] === undefined) return
+
+    if( data.length===0 || data[0] === undefined){
```
Comment (mohitkhatri06) [inline src/components/Charts/AssetsDepreciationsChart.jsx:55]: when I use a test user which does not have any assets entry.
image

Comment (ivangarcilazo) [reply src/components/Charts/AssetsDepreciationsChart.jsx:55]: I tried different ways, and it always tells me that it has no assets, I implemented one more thing that should prevent that from happening. If you want I'll show you, it works fine for me when the user doesn't have depreciations yet.

**Comment by mohitkhatri06** [comment]
for reacter credential did you checked? its not displaying chart for that also now.
image

**Comment by mohitkhatri06** [comment]
I saw the calculations in code, I have few doubts about that, can you share how you approached to the depreciation data.

**Comment by ivangarcilazo** [comment]
I used gpt chat to know what data I needed, and I was also guided with information about the economy that I found on the internet, I looked for things like: "how to calculate the depreciation of assets" and always gave the following formula: "To calculate the accumulated depreciation, the total cost of the asset is divided by its estimated useful life and the result is multiplied by the Although I take many other references.

**Review by mohitkhatri06 — DISMISSED** [review]

**Review by mohitkhatri06 — DISMISSED** [review]

**Review by mohitkhatri06 — APPROVED** [review]

**Review by mohitkhatri06 — APPROVED** [review]

**Review by mohitkhatri06 — APPROVED** [review]

**Review by mohitkhatri06 — DISMISSED** [review]

File: src/Router.jsx [hunk 3]
```diff
@@ -74,8 +74,6 @@ const RouterComponent = () => {
                 <>
                   <Header />
                   <IncomeStatement Context={Context} />
-                  <Footer />
-                  <SupportPopUp Context={Context} />
```
Comment (mohitkhatri06) [inline src/Router.jsx:78]: I looked into the master branch this was there.

**Comment by ivangarcilazo** [comment]
Mohit, I did a merge from master, and made the changes you indicated. This shouldn't cause any problems, I've been checking to avoid this.

**Review by mohitkhatri06 — APPROVED** [review]

**Comment by bharathkumarchowdary** [comment]
> Hi Mohit! I sent you a message on Microsoft teams about a question I have.

@ivangarcilazo , @mohitkhatri06 ,
Am I missing something here?

**Comment by ivangarcilazo** [comment]
Not really, it's a doubt about some conflicts of this branch, it let me do squash and merge but not rebase and merge, now I have to figure out how to solve it, there Mohit gave me a hand.

**Pull request merged**